### DISCUSSION TITLE
Generic Service Stack

### DISCUSSION THREAD

**Discussion by darshan-yadav** [aws/aws-cdk#15509] [Q&A]
I want to create a generic stack that can deploy services for a different name. Use case if something like I want to create a service for each customer. So I have parameterized the service name, task def.  But when I deploy the second service it deletes the first one. How can I avoid that without creating a different stack?

**Comment by peterwoodworth** [comment]
Could you please provide more info about what "services" are specifically referring to, and what each stack consists of?
